ihatemoney/conf-templates/apache-vhost.conf.j2

```diff
@@ -1,8 +1,7 @@
 <VirtualHost *:80>
     ServerAdmin admin@example.com      # CUSTOMIZE
     ServerName ihatemoney.example.com  # CUSTOMIZE
-
-    WSGIDaemonProcess ihatemoney user=www-data group=www-data threads=5 python-path={{ pkg_path }} {% if venv_path %}python-home={{ venv_path }}{% endif %}
+    WSGIDaemonProcess ihatemoney user=www-data group=www-data threads=5 python-path={{ pkg_path }} {% if bin_path %}python-home={{ bin_path }}{% endif %}
```

Comment:
I think this simple « drop-in » replacement might not be good, as `venv_path` does not point the same thing as `bin_path` (to my understanding).

Reply:
Yeah, I'm not sure what this should point to, actually!

We probably should add another set of variables when à venv is in use.

Reply:
Bump :)

I think we break some use case here (`generate-config apache` within venv), so IMHO, better fixing that before merging :)

Reply:
yeah, sorry, I lack some time these days with the opening of the brewery. I'll give it a look when I can, but don't hesitate to take over otherwise!

Reply:
@almet I did :-). I PR-ed your PR (crazy !), see #359 I let you the care to merge it and rebase.

*Hint: the ChangeLog might have to be fixed since 2.1 has been released since.*

Not that this should work but is untested on a real apache (I do not use apache), . @0livd, as the official reference ihm-with-apache2 user if you have some time to test the apache2 setup outputed by `ihatemoney ihatemoney generate-config apache-vhost.conf`, feel free :-).